src/server/game/Handlers/SpellHandler.cpp

```diff
@@ -640,7 +640,7 @@ void WorldSession::HandleMirrorImageDataRequest(WorldPacket& recvData)
         data << uint8(player->GetFacialStyle());
         data << uint32(player->GetGuildId());
 
-        static EquipmentSlots const itemSlots[] =
+        static std::array const itemSlots =
```

Comment:
This looks like `auto` to me (array of what?), since we only just enabled c++17, not sure how to proceed

Reply:
It's technically auto indeed. If we don't specify type nor size the compiler will do what it was designed to do and deduct the type for us along with the size.
https://www.learncpp.com/cpp-tutorial/6-15-an-introduction-to-stdarray/ That's the rules apparently, i could specify the size there as well.